prime-router/src/test/kotlin/ReportTests.kt

```diff
@@ -271,8 +273,60 @@ class ReportTests {
     }
 
     @Test
-    fun `test patient age validation`() {
+    fun `test patient age deidentification`() {
+        val patientAgeSchema = Schema(
+            name = "patientAgeSchema",
+            topic = "test",
+            elements = listOf(
+                Element("patient_age", pii = true),
```

Comment:
In the schema, patient_age has `pii = false`, I'm not sure if this makes any difference.

Reply:
Since patient age and patient zip aren't always PII they should come through sometimes. Marking them as PII would block them entirely. The logic in the deidentify() function looks for both fields specifically.